documentation/common/address-space.adoc

```diff
@@ -127,3 +127,52 @@ The multicast address type is a scalable direct address for sending messages to
 * standard
 
 // address.multicast.plan.inmemory.shortDescription:Configures router network with multicast address
+
+[[brokered_address_space]]
+=== Brokered Address Space
+
+The brokered address space is designed to support broker-specific features, at the cost of limited
+scale in terms of the number of connections and addresses. This address space supports JMS
+transactions, message groups etc. The brokered address space is backed by a broker, and stores
+persisted messages on disk if required.
+
+Clients connect and send and receive messages in this address space using the AMQP protocol.
+
+==== Address types
+
+* queue
+* topic
+
+[[brokered-queue]]
+==== Queue
+
+The queue address type is a store-and-forward queue. This address type is appropriate for
+implementing a distributed work queue, handling traffic bursts, and other use cases where you want
+to decouple the producer and consumer. A queue in the brokered address spaces supports selectors,
+message groups, transactions and other JMS features. If the queue is a high volume queue and these
+semantics are not needed, have a look at the `standard` address space `queue` type.
+
+[[brokered-queue-plans]]
+===== Queue Plans
+
+* standard
+
+[[standard-queue-plan]]
+====== Standard
+The standard queue plan deploys a queue in a broked instance shared with other addresses in the
```

Comment:
Only issue with this is that it sounds like other plans might not share the broker.